lib/instrumentation/transaction.js

```diff
@@ -143,6 +143,7 @@ Transaction.prototype.startSpan = function (name, type) {
     return null
   }
   if (this._builtSpans >= this._agent._conf.transactionMaxSpans) {
+    this._agent._metrics.getOrCreateCounter('agent.spans.dropped').inc()
```

Comment:
We decided to delay the introduction of `agent.*` metrics, so this line should just be removed